toolchain/check/import_ref.cpp

```diff
@@ -2073,13 +2074,46 @@ static auto AddImplDefinition(ImportContext& context,
   }
 }
 
+static auto ResolveSpecificInterface(ImportRefResolver& resolver,
```

Comment:
The naming convention we use elsewhere would call this `GetLocalSpecificInterface`.